test/dotnet-watch.FunctionalTests/ProjectToolScenario.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.DotNet.Watcher.FunctionalTests
+{
+    public class ProjectToolScenario: IDisposable
+    {
+        private const string NugetConfigFileName = "NuGet.Config";
```

Comment:
Careful with capitalization here, `NuGet.config` is the actual correct string, could cause issues on Unix